client/jetpack-cloud/sections/agency-signup/agency-signup-form/index.tsx

```diff
@@ -61,6 +65,7 @@ export default function AgencySignupForm() {
 					country: payload.country,
 					postal_code: payload.postalCode,
 					state: payload.state,
+					referrer: referrer,
```

Comment:
Since `referrer` is being captured from the outer scope it should be added to the `deps`array that is the second parameter of `useCallback`.

Reply:
Alternatively, `referrer` could be grabbed from `payload.referrer` instead. I think that's more consistent so that what is sent to `recordTracksEvent` is guaranteed to match what is sent to `createPartner.mutate`.

Reply:
Thank you @robertf4! I'll go with `payload.referrer` as that was added in an earlier iteration when the referrer was passed as a prop, and it sounds like that is the best option.